--- origsrc/libjpeg-turbo-2.1.0/CMakeLists.txt	2021-04-24 03:29:40.000000000 +0900
+++ src/libjpeg-turbo-2.1.0/CMakeLists.txt
@@ -206,6 +206,9 @@
   set(ENABLE_SHARED 1)
 endif()
 
+option(WITH_JPEGINT_H "Dare to install jpegint.h" FALSE)
+boolean_number(WITH_JPEGINT_H)
+
 # Explicitly setting CMAKE_POSITION_INDEPENDENT_CODE=FALSE disables PIC for all
 # targets, which will cause the shared library builds to fail.  Thus, if shared
 # libraries are enabled and CMAKE_POSITION_INDEPENDENT_CODE is explicitly set
@@ -1516,6 +1519,12 @@
   ${CMAKE_CURRENT_SOURCE_DIR}/jpeglib.h
   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
+if(WITH_JPEGINT_H)
+  install(FILES
+    ${CMAKE_CURRENT_SOURCE_DIR}/jpegint.h
+    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+endif()
+
 include(cmakescripts/BuildPackages.cmake)
 
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/cmakescripts/cmake_uninstall.cmake.in"
